### ISSUE TITLE
Please make it clearer, which Linux (Kernel) versions are supported for the Azure Monitor Extension

### ISSUE THREAD

**Issue by robinmanuelthiel** [microsoft/azure_arc#100]
**Is your feature request related to a problem? Please describe.**
I ran into the following issue: I created an Ubuntu VM in AWS and tried to install the Azure Monitor Extention onto it via the Azure Portal, which at first failed without a clear error message. After noticing that Python 2 needs to be installed on the VM (which was not part of the error message), I was able to install the `OMSAgentForLinux` extension, but the `DependencyAgentLinux` extension still failed with a weird `Conflict` error message.

Only after trying to [install the Dependency Agent manually](http://thewindowsupdate.com/2020/02/25/linux-ubuntu-vms-in-non-responsive-state-for-service-map-agent-02-25-resolved/), I got an informative error message: 

```
Unsupported kernel version: 5.3.0-1032-aws
```

As this is the default kernel on AWS Ubuntu VMs, many users might run into this. The documentation does link [supported Linux version](https://docs.microsoft.com/en-us/azure/azure-arc/servers/agent-overview#supported-operating-systems) but do not make clear, that **Ubuntu might be supported, but the AWS kernel is not**.

**Describe the solution you'd like**
Document, that AWS Ubuntu Kernel is not supported.

**Describe alternatives you've considered**
-

**Additional context**
Copy of https://github.com/MicrosoftDocs/azure-docs/issues/61021

**Comment by robinmanuelthiel** [comment]
Same with Amazon Linux 2 by the way, which also [should be supported](https://docs.microsoft.com/en-us/azure/azure-arc/servers/agent-overview#supported-operating-systems)

```
Unsupported kernel version: 4.14.186-146.268.amzn2.x86_64
```

**Comment by robinmanuelthiel** [comment]
Tried several things now, even switching the kernel version of a GCP VM to the default one, but can't get the Monitoring Agent installed.

![image](https://user-images.githubusercontent.com/5874465/90436438-f6673600-e0d0-11ea-99f4-7e03a72f35ad.png)

**Comment by likamrat** [comment]
@robinmanuelthiel please refer to [Enable Azure Monitor for VMs overview](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/vminsights-enable-overview#supported-operating-systems). 
As you can see, the supported version for Ubuntu 18.04 is 5.3.0-**1020** and not 1030. 

Thx for pointing that out, we will update the doc accordingly and will also reach out to the product group to let them know about this issue. 

Closing this issue for now
